### PR TITLE
Bugs/7983

### PR DESCRIPTION
**Pull request by sslaws**
## Status
Ready

## Fixes or Implements [AB#7983](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7983), [AB#7988](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7988), [AB#7985](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7985), [AB#7987](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7987), [AB#7989](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7989)
- [ ] Enhancement
- [X] Bug
- [ ] Documentation

## Description:
Fixed [AB#7983](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7983) - Email Service accepts invalid emails (whitespace)
Fixed [AB#7988](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7988) - Email Sent DateTime not set
Fixed [AB#7985](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7985) - Period in ToS template is in anchor
Fixed [AB#7987](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7987) - Closure date calculated wrong
Fixed [AB#7989](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7989) - Deletion should be calculated in hours not days
## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### Steps to Reproduce:
N/A

### UI Changes
No

### Browsers Tested
- [ ] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.
Manual testing 

## Notes/Additional Comments
N/A

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_360)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_360)

File: Apps/JobScheduler/src/Jobs/CloseAccountJob.cs [hunk 1]
```diff
@@ -80,7 +80,7 @@ public class CloseAccountJob
         [DisableConcurrentExecution(ConcurrencyTimeout)]
         public void Process()
         {
-            DateTime deleteDate = DateTime.Now.Date.AddDays(this.daysBeforeDeletion);
+            DateTime deleteDate = DateTime.UtcNow.Date.AddDays(this.daysBeforeDeletion);
```
Comment (FuriousLlama) [inline Apps/JobScheduler/src/Jobs/CloseAccountJob.cs]: Is the daysBeforeDeletion variable comming from configuration? if so I think it should be in hours.

Comment (sslaws) [reply Apps/JobScheduler/src/Jobs/CloseAccountJob.cs]: I've updated this to hours

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_360)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_360)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_360)

**Pull request merged**